src/js/tabs/led_strip.js

```diff
@@ -344,33 +344,19 @@ TABS.led_strip.initialize = function (callback, scrollPosition) {
 
                         // Get function & overlays or current cell
                         TABS.led_strip.directions.forEach(function(letter) {
-                            var className = '.dir-' + letter;
+                            const className = '.dir-' + letter;
                             if ($(that).is(className)) {
                                 directionsInSelection.push(className);
                             }
                         });
-
-                        TABS.led_strip.baseFuncs.forEach(function(letter) {
```

Comment:
Where did these go?

Reply:
Not sure why I removed them. There was no change in functionality. To be sure added them back and there seems no difference. This tab is too complicated and was the hardest to fix.